chain/chain/src/chain.rs

```diff
@@ -4030,3 +4038,21 @@ pub fn collect_receipts_from_response(
         receipt_proof_response.iter().flat_map(|ReceiptProofResponse(_, proofs)| proofs),
     )
 }
+
+#[derive(Message)]
+#[rtype(result = "()")]
+pub struct StatePartsMessage {
```

Comment:
Could you rename these two structs to `ApplyStatePartsRequest` and `ApplyStatePartsResponse`? I think that explains what these messages imply better.